packages/selection/src/containers/SelectionContainer.js

```diff
@@ -169,6 +169,12 @@ export class SelectionContainer extends ControlledComponent {
 
   onFocusSelectionModelChange = ({ newSelection }) => {
     const focusedKey = this.indexKeyMap[newSelection];
+    const activeElement = document.activeElement;
```

Comment:
Could we use the `activeElement` `dom-helper` utility here for cross-browser support? https://www.npmjs.com/package/dom-helpers

This helps cover the weird use-cases in IE11 with returning from another tab (probably some other things as well).